civihr_default_theme/assets/partials/myleave.html

```diff
@@ -0,0 +1,50 @@
+<div>
+
+  <div class="button-container">
+
+    <div class="btn-group">
+      <button type="button" class="chr_action dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
```

Comment:
Don't use `.chr_action` or **any** of these custom components that could be traced back to a Bootstrap component instead. So instead of `.chr_action`, this is a `.btn.btn-primary`